mne/inverse_sparse/tests/test_mxne_inverse.py

```diff
@@ -170,6 +170,23 @@ def test_mxne_vol_sphere():
     assert_true(isinstance(stc, VolSourceEstimate))
     assert_array_almost_equal(stc.times, evoked_l21.times, 5)
 
+    # Compare orientation obtained using fit_dipole and mixed_norm
+    # for a simulated evoked
+    pos = src[0]['rr'][stc.vertices[0]]
+    evoked_dip = mne.simulation.simulate_evoked(fwd, stc, info, cov, nave=1e9)
+
+    dip_mxne = mixed_norm(evoked_dip, fwd, cov, alpha=80,
+                          n_mxne_iter=1, maxit=30, tol=1e-8,
+                          active_set_size=10, return_as_dipoles=True)
+
+    amp_max = [np.max(d.amplitude) for d in dip_mxne]
+    dip_mxne = dip_mxne[np.argmax(amp_max)]
+    assert_true(dip_mxne[0].pos[0] in pos)
```

Comment:
this does not pass on master but the assert below on ori does.

so this test is not enough to demonstrate the bug fix on orientation
estimation.